src/api/java/leaf/cosmere/api/Constants.java

```diff
@@ -83,12 +82,17 @@
 		public static final String KEY_MANIFESTATION_NEXT = "key.cosmere.powers.next";
 		public static final String KEY_MANIFESTATION_PREVIOUS = "key.cosmere.powers.previous";
 
+		public static final String KEY_MANIFESTATION_USE_ACTIVE = "key.cosmere.powers.use";
+
 		public static final String KEY_DEACTIVATE_ALL_POWERS = "key.cosmere.powers.toggle";
 		public static final String KEY_MANIFESTATION_MENU = "key.cosmere.powers.menu";
 
 		public static final String KEY_ALLOMANCY_PUSH = "key.cosmere.allomancy.push";
 		public static final String KEY_ALLOMANCY_PULL = "key.cosmere.allomancy.pull";
 
+		public static final String KEY_SANDMASTERY_LAUNCH = "key.cosmere.sandmastery.launch";
```

Comment:
We might end up moving mod specific constants to the mods themselves, but this is fine for now